engine/src/main/java/org/terasology/logic/players/LocalPlayerSystem.java

```diff
@@ -222,28 +227,21 @@ private void standPlayer(EntityRef entity) {
      */
     private void stopAutoMove() {
         isAutoMove = false;
+        List<Input> inputs = bindsConfig.getBinds(new SimpleUri("engine:forwards"));
+        Input forwardKey = inputs.get(0);
```

Comment:
It's not hard crashing, but I'll put a check for an empty input (plus the new constructor) and push it soon!